test/environment_unittest.cc

```diff
@@ -1,10 +1,9 @@
 #include "../src/environment.h"
```

Comment:
[Unrelated to this PR] Do we have any formatter enforced? I guess not because otherwise it will complain about `#include` not sorted?

Reply:
I don't use any formatter, but it would be nice if we had one.